SPECS/sqlite/sqlite.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "sqlite-autoconf-3440000.tar.gz": "b9cd386e7cd22af6e0d2a0f06d0404951e1bef109e42ea06cc0450e10cd15550"
+  "version-3.44.5.tar.gz": "24918af2f9cb0d2cd103b88212ef28deedf4dca2033b986792a8b7ce8fcc614c"
```

Comment:
This file name is not acceptable for tarball upload to the lookaside cache.   It should be something like sqlite-3.44.5.tar.gz.

All tarballs are stored in a flat directly and could clash with some other source rpm.  The upload dialog gives you the opportunity to rename the tarball before upload.